channels.yaml

```diff
@@ -8,8 +8,12 @@ appDefaults:
         defaultVersion: '1.22.x'
       - appVersion: '>= 2.6.5-0 < 2.6.7-0'
         defaultVersion: '1.23.x'
-      - appVersion: '>= 2.6.7-0 < 2.7.100-0'
+      - appVersion: '>= 2.6.7-0 < 2.7.0-0'
         defaultVersion: '1.24.x'
+      - appVersion: '>= 2.7.0-0 < 2.7.2-0'
+        defaultVersion: '1.24.x'
```

Comment:
Same comment as for `channels-rke2.yaml`.

Reply:
updated it.